src/sentry/static/sentry/app/components/customResolutionModal.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import jQuery from 'jquery';
+import Modal from 'react-bootstrap/lib/Modal';
+import underscore from 'underscore';
+
+import TimeSince from './timeSince';
+import Version from './version';
+
+import {Select2FieldAutocomplete} from './forms';
+import {t} from '../locale';
+
+export default React.createClass({
+  propTypes: {
+    onSelected: React.PropTypes.func,
+    onCanceled: React.PropTypes.func,
+    orgId: React.PropTypes.string.isRequired,
+    projectId: React.PropTypes.string.isRequired,
+    show: React.PropTypes.bool
+  },
+
+  getInitialState() {
+    return {version: ''};
+  },
+
+  componentDidUpdate(prevProps, prevState) {
+    if (this.props.show) {
```

Comment:
Do you not want to see if there was a change between `prevProps` and `this.props`? Cause `componentDidUpdate` can trigger repeatedly in succession where `this.props.show` doesn't actually change (literally any prop/state change will trigger it).

(Not that this is doing anything heavy.)